content/en/docs/contribute/new-content/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Contributing new content overview
+linktitle: Overview
+content_template: templates/concept
+main_menu: true
+weight: 5
+---
+
+{{% capture overview %}}
+
+This section contains information you should know before contributing new content.
+
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Contributing basics
+
+- Write Kubernetes documentation in Markdown and build the Kubernetes site using [Hugo](https://gohugo.io/).
+- The source is in [GitHub](https://github.com/kubernetes/website). You can find Kubernetes documentation at `/content/en/docs/`. Some of the reference documentation is automatically generated from scripts in the `update-imported-docs/` directory.
+- [Page templates](/docs/contribute/style/page-templates/) control the presentation of documentation content in Hugo.
+- In addition to the standard Hugo shortcodes, we use a number of [custom Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/) in our documentation to control the presentation of content.
+- Documentation source is available in multiple languages in `/content/`. Each language has its own folder with a two-letter code determined by the [ISO 639-1 standard](https://www.loc.gov/standards/iso639-2/php/code_list.php). For example, English documentation source is stored in `/content/en/docs/`.
+  - For more information about contributing to documentation in multiple languages or starting a new translation, see [localization](/docs/contribute/localization).
+
+## Before you begin {#before-you-begin}
+
```

Comment:
Do you need `Before you begin` (empty h2)

Reply:
It groups the following h3's – which are all things you need to do `Before you begin`.

Reply:
Your call, I checked: https://developers.google.com/style/headings